content/administration/odoo_sh/getting_started/first_module.rst

```diff
@@ -502,7 +519,8 @@ Stage and commit your changes:
 
   $ git add requirements.txt
   $ git add my_module
-  $ git commit -m "[IMP] my_module: automatically remove special chars in my_module.my_module name field"
+  $ git commit -m "[IMP] my_module: automatically remove special chars in my_module.my_module name
+    field"
```

Comment:
This can stay on one line ;).
In some cases, strictly respecting a line length of max 100 characters modifies is not necessary, or even modifies the generated page.
Here, it's in a code-block directive, and it matters to keep the whole command on one line ;).

Before your changes:
![image](https://user-images.githubusercontent.com/19364378/207036282-47276722-0dfc-41c5-84d2-d454b364c13b.png)

With your changes:
![image](https://user-images.githubusercontent.com/19364378/207036423-5b18e833-010c-4347-8abd-951c96028ecc.png)

Reply:
You didn't remove your change on this line here :eyes:

Reply:
@toaa-odoo you forgot this

Reply:
My bad, I changed it and then had to reset --hard because I forgot to pull, and forgot that change.